store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/DemoMiniAccumuloStore.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016-2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.accumulostore;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import uk.gov.gchq.gaffer.store.StoreException;
+import uk.gov.gchq.gaffer.store.StoreProperties;
+import uk.gov.gchq.gaffer.store.schema.Schema;
+
+import java.io.File;
+
+public class DemoMiniAccumuloStore extends AccumuloStore {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DemoMiniAccumuloStore.class);
+    private static AccumuloTestClusterManager accumuloTestClusterManager;
+    private static final AccumuloProperties ACCUMULO_PROPERTIES = new AccumuloProperties();
+
+    static {
+        ACCUMULO_PROPERTIES.setStoreClass(MiniAccumuloStore.class);
+        ACCUMULO_PROPERTIES.setZookeepers("aZookeeper");
```

Comment:
It works because the properties instance gets updated by the AccumuloTestClusterManager with the actual zookeeper details for the cluster it has started.
That is why I made accumuloTestClusterManager accept a StoreProperties object rather than a file reference or name: it allows the zookeeper details to be passed back seamlessly.

Reply:
Okay in that case is it worth removing it to avoid confusion?

Reply:
I shall remove it.